playbooks/maas-verify.yml

```diff
@@ -22,7 +22,7 @@
       file:
         path: "{{ workspace_logs }}"
         state: "directory"
-        mode: "0755"
+        mode: "1777"
```

Comment:
Maybe it's best to just leave the mode out, so that it doesn't change it if the dir already exists.  The problem w/ using `1777` here is that we don't want some arbitrary dir. to have that perm if the deployer sets `workspace_logs`.

Reply:
Done